index.html

```diff
@@ -0,0 +1,637 @@
+            <hr>
+            <img src="https://d.newsweek.com/en/full/1574439/dan-caldwell.png?w=63&h=63&f=9112a861ea096c0b55f6d98918261dad" alt="">
+            <p class="opinion-p">Coronavirus Is Exposing Trump's 'Strong Economy' for the Hoax It Is</p>
+            <span>BY ANGELA HANKS</span>
+            <hr>
+            <img src="https://d.newsweek.com/en/full/101658/jennifer-weiss-wolf.png?w=63&h=63&f=0b82c369922ff03b0e424910c2ec3167" alt="">
+            <p class="opinion-p">Don't Panic Over Coronavirus. That's Just What the President's Enemies Want</p>
+            <span>BY CHARLIE KIRK</span>
+            <hr>
+            <img src="https://d.newsweek.com/en/full/83499/newt-gingrich.png?w=63&h=63&l=48&t=55&f=d8b4f8dfaad8a5f0f71bffcc6c0d6a77" alt="">
+            <p class="opinion-p">Coronavirus and In-Person Voting Don't Mix. We Must Prepare for Digitized Elections</p>
+            <span>BY LEEZA GARBER</span>
+            <hr>
+            <img src="https://d.newsweek.com/en/full/83499/newt-gingrich.png?w=63&h=63&l=48&t=55&f=d8b4f8dfaad8a5f0f71bffcc6c0d6a77" alt="">
+            <p class="opinion-p">Trump Is Hurting the Economy by Linking Coronavirus to Immigration</p>
+            <span>BY IBRAHIM ALHUSSEINI</span>
+            <hr>
+
+            <h4>LATEST NEWS</h4>
+            <div class="congress">
+              <img src="https://d.newsweek.com/en/full/1574961/don-young.jpg?w=95&h=95&f=e61b82af5d45f0beeffdb3b7894d6fca" alt="">
+              <p>Congressman Attended NRA Fundraiser After Skipping COVID-19 Relief Vote</p>
+            </div>
+            <hr>
+            <div class="congress1 d-flex justify-content-center">
+              <img src="https://d.newsweek.com/en/full/1574927/candle.jpg?w=95&h=95&f=9041fb7e8d050adba66d063a53b19abc" alt="">
+              <p>My Family Gave Up Sitting Shiva to Fight Coronavirus</p>
+            </div>
+            <div class="img">
+              <img src="img/21.png" alt="">
+            </div>
+            <div class="img">
+              <img src="img/22.png" alt="">
+            </div>
+            <div class="img">
+              <img src="img/23.png" alt="">
+            </div>
+            <div class="img">
+              <img src="img/24.png" alt="">
+            </div>
+
+          </div>
+        </div>
+    </section>
+
+    <section class="magazine">
+      <div class="container">
+        <div class="row n-magazine">
+          <div class="mag-heading">
+            <h4> <img src="images/l.png" alt=""> IN THE MAGAZINE</h4>
+            <hr>
+          </div>
+
+          <div class="col-lg-3  mag col-md-3 ">
+            <img src="https://d.newsweek.com/en/full/1574205/march-20-27-2020-cover.jpg?w=360&h=480&q=90&f=f32c43b8e7ecd7514c13923e4edef489" alt="">
+          </div>
+          <div class="col-lg-3 mag-1 col-md-3">
+            <img src="images/e.png" alt="">
+            <a href="#">Is Bernie Sanders Right About Medicare For All?</a>
```

Comment:
- [ ] These types of titles should be bold and have a red underline hover. They repeat throughout the page, so consider creating a class for them.